app/components/candidate_interface/application_choice_item_component.rb

```diff
@@ -40,4 +41,14 @@ def view_application_path
       candidate_interface_course_choices_course_review_path(application_choice.id)
     end
   end
+
+  def show_decline_by_default_text?
+    application_choice.offer? && Time.zone.now.between?(timetable.apply_deadline_at, decline_by_default_at)
```

Comment:
You can use your new method here:
```suggestion
    application_choice.offer? && timetable.between_apply_deadline_and_decline_by_default?)
```